front/lib/api/tables.ts

```diff
@@ -0,0 +1,417 @@
+import type {
```

Comment:
This is really just moved code. The code was originally in the (private AP) endpoint file. We now need this in the public API as well, so instead of duplicating the whole thing I am moving it here.